docs/docs-content/vertex/system-management/account-management/email.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: "Email Address"
+title: "Email Address"
+description: "Update and manage the email address of the admin user."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["vertex", "management", "account", "credentials"]
+keywords: ["self-hosted", "vertex"]
+---
+
+You can manage the credentials of the admin user by logging in to the system console. Updating or changing the email address of the admin user requires the current password.
+
+## Email Address
```

Comment:
Is this H2 heading required? It's the same as the title of the page

Reply:
Good point